jmix-search/search/src/main/java/io/jmix/search/index/impl/SearchSettingsComparator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.search.index.impl;
+
+import org.elasticsearch.common.settings.Settings;
+import org.springframework.stereotype.Component;
+
+@Component("zndfl_SearchSettingsComparator")
```

Comment:
zndfl?

Reply:
Fixed. And component names to other classes were added.